.github/workflows/build-test-e2e.yml

```diff
@@ -9,9 +9,9 @@ env:
   PW_ALL_BLOBS_DIR: all-blob-reports
 jobs:
   build:
-    # Run in Sage repo on develop or release- branches
+    # Run in Sage repo on release- branches
     # and on all branches in user-owned forks
-    if: ${{ github.ref_name == 'develop' || startsWith(github.ref_name, 'release-') || github.actor == github.repository_owner }}
+    if: ${{ startsWith(github.ref_name, 'release-') || github.actor == github.repository_owner }}
```

Comment:
Only run e2e workflow for `release-**` branches in Sage repo, so macOS runners are available for other teams' workflows